docs/cmdlet-example/command-line-simple-example.md

```diff
@@ -0,0 +1,224 @@
+# Creating a cross-platform binary module with the .NET Core command-line interface tools
+
+This example uses the [.NET Core command-line interface tools][dotnet-cli] (aka
+`dotnet CLI`) to demonstrate how to create a binary module that is portable across operating
+systems supported by `PowerShell Core` as well as `Windows PowerShell` version 3 and higher.
+
+Because the binary module's assembly will be created as a `.NET Standard 2.0 class library`,
+the same assembly can be imported into both PowerShell Core and Windows PowerShell.
+This means you do not have to build and distribute separate assemblies that target these two
+different implementations of PowerShell.
+
+## Prerequisites
+* PowerShell Core and/or Windows PowerShell
+
+  For this example, you can use any operating system that is supported by PowerShell Core.
+  To see if your operating system is supported and to get instructions on how to install
+  PowerShell Core on your operating system, see the [Get PowerShell][pscore-os] topic in
+  the PowerShell repo's [README.md][readme] file.
+
+  Note: On Windows 10 Anniversary Update or higher, you can use the [Windows Subsystem for
+  Linux][wsl] (WSL) console to build the module. In order to import and use the module, you'll need
+  to install `PowerShell Core` for the distribution and version of Linux you're running.
+  You can get that version info by running the command `lsb_release -a` from the WSL console.
+
+* .NET Core 2.0 SDK
+
+  Download and install the [.NET Core 2.0 SDK][net-core-sdk] for your operating system.
+  It is recommended that you use a package manager to install the SDK on Linux.
+  See these [instructions][linux-install] on how to install the SDK on Linux.
+  Be sure to pick your distribution of Linux e.g. RHEL, Debian, etc to get the
+  appropriate instructions for your platform.
+
+## Create the .NET Standard 2.0 Binary Module
+
+1. Verify you are running the 2.0.0 version of the dotnet CLI.
```

Comment:
I think this should be used for all occurrence: `dotnet` CLI

Reply:
Are you saying to code escape dotnet .e.g. `dotnet` everywhere it is used in conjunction with CLI? If so, I can do that.

BTW I had more stuff code-escaped at one point but then a I was reading it, the escaping started to look too busy.  I found I was using it for "code" and to draw attention particular nouns for those who skim.

Reply:
I use code escaping for commands, files names, and assembly names as a general rule. Other things like the SDK name might be a bit much. for this, `dotnet` is a command and "CLI" is a description of that command. I think "`dotnet` CLI" is probably the best approach.